mastercraft.cmd

```diff
@@ -1515,7 +1561,8 @@ gather.material:
                if "%order.pref" = "runestone" then var work.material basic
                if "%order.pref" = "sphere" then var work.material small
                evalmath %get.mat.item.count %%get.mat.item.count - 1
-               gosub GET %work.material %get.mat
+			   # Kzin added "from my %main.storage" to prevent grabbing things from the bin.
+               gosub GET %work.material %get.mat from my %main.storage

```

Comment:
The comment says it all. I had to say "get my totem" instead of "get totem" or it would pull the crap out of the trash I was trying to throw away, as an example.